masterlist.yaml

```diff
@@ -17642,6 +17642,11 @@ plugins:
         name: 'Carriage and Ferry Travel Overhaul'
     after: [ 'Immersive Citizens - AI Overhaul.esp' ]
     msg:
+      - <<  *alsoUseX
+        subs: [ '[Carriage and Ferry Travel Overhaul - Fixes and Winterhold](https://www.nexusmods.com/skyrimspecialedition/mods/40651?tab=description)' ]
+      - <<  *alsoUseX
+        subs: [ '[Carriage and Ferry Travel Overhaul - Companions Radiant Quest Fix](https://www.nexusmods.com/skyrimspecialedition/mods/52353?tab=description)' ]
```

Comment:
Please shorten the url as much as possible & add a trailing slash at the end of it, so the url should look like this:
`https://www.nexusmods.com/skyrimspecialedition/mods/52353/`